iquip/apps/dataviewer.py

```diff
@@ -967,16 +977,23 @@ def _toggleSync(self, checked: bool):
         """
         if checked:
             self.synchronize()
-        elif self.thread is not None:
+            return
+        try:
             self.thread.stop()
```

Comment:
When can a `RuntimeError` be occurred?

Reply:
Somehow the `thread` is deleted by `deleteLater()` and `thread.stop()` is called.